src/structlog/stdlib.py

```diff
@@ -466,6 +466,8 @@ class ProcessorFormatter(logging.Formatter):
         ``True`` to keep it on the :class:`logging.LogRecord`. (default: False)
     :param bool keep_stack_info: Same as *keep_exc_info* except for Python 3's
         ``stack_info``.
+    :param logger: Logger which we want to push through the structlog processor
```

Comment:
* Please add the default value
* and a `.. versionadded` tag. 
* wrap `structlog` in double ticks – that's our corporate brand. ;)
* I feel like this needs more explanation what it is actually good for? Also maybe more info in the stdlib docs so people know (how) to actually use it?
* Bonus points for adding the default to keep_stack_info too. :)